examples/platform/silabs/efr32/rs911x/sl_wlan_config.h

```diff
@@ -0,0 +1,110 @@
+/*

```

Comment:
This looks like a highly duplicated file from 
`examples/platform/silabs/SiWx917/SiWx917/sl_wlan_config.h`

why can't they be merged